infra/indexers/src/usdc.ts

```diff
@@ -0,0 +1,26 @@
+import { ec, hash } from "./deps.ts";
+
+export const USDC_ADDRESS =
+  "0x53c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8";
+
+export const USDC_DECIMALS = 6;
+
+/**
+ * Computes the storage location of the balance of an address.
+ *
+ * @param address The address.
+ * @returns The storage location.
+ */
+export function balanceStorageLocation(address: string) {
+  const addressBound = 2n ** 251n - 256n;
+
+  let hashed = hash.getSelectorFromName("ERC20_balances");
+  hashed = ec.starkCurve.pedersen(hashed, address);
+
+  let location = BigInt(hashed);
+  while (location > addressBound) {
+    location -= addressBound;
+  }
```

Comment:
what's happening here ?

Reply:
It's a mod operator that does nothing in most cases. I can change it to `%` if it's more clear (I don't think performance is much different).

Reply:
the range is `0 <= x < 2 ** 251` for `StorageAddress`
I don't think a while loop is needed as the last step is pedersen which is `<P` so there should only be a small surplus if we have one

Reply:
let's do `if`